src/server/src/main/java/io/cassandrareaper/resources/RepairRunResource.java

```diff
@@ -608,7 +608,9 @@ public Response getRepairRunsForCluster(
       @QueryParam("limit") Optional<Integer> limit) {
 
     LOG.debug("get repair run for cluster called with: cluster_name = {}", clusterName);
-    final Collection<RepairRun> repairRuns = context.storage.getRepairRunsForCluster(clusterName, limit);
+    final Collection<RepairRun> repairRuns = context
+        .storage
+        .getRepairRunsForClusterPrioritiseRunning(clusterName, limit);
```

Comment:
You need to do the same on line 665. It's the method that lists repair runs throughput all registered clusters.
Currently it's still using the old `getRepairRunsForCluster()`  method which isn't optimized to prioritize running repairs.

Reply:
The problem is that on line 665, it is adding the runs from ALL clusters into a list. I'll need to re-sort the runs so that the statuses are in the right order irrespective of cluster. Once you let me know the precise way you want the sorting done, I might just encapsulate it in a function so that we aren't duplicating that logic in multiple places.

Reply:
Right, you'll need to merge lists from different clusters and re-sort to apply the limits.
FYI, repair ids are timeuuids, which makes it possible to apply the sorting on them directly using their time component.

Reply:
This appears to be done and working now, although the cucumber test doesn't test against multiple clusters. I'm not sure if I should add a multi-cluster test to confirm that this endpoint does indeed work?